lib/MusicBrainz/Server/Edit/Artist/Edit.pm

```diff
@@ -193,9 +193,39 @@ around allow_auto_edit => sub {
     return 0 if exists $self->data->{old}{end_area_id} &&
         defined($self->data->{old}{end_area_id}) && $self->data->{old}{end_area_id} != 0;
 
-    return 0 if $self->data->{new}{ipi_codes};
+    if (defined $self->data->{new}{ipi_codes}) {
+        # If there's already IPIs for the artist, not an autoedit
+        if (defined $self->data->{old}{ipi_codes} &&
+            scalar @{$self->data->{old}{ipi_codes}} != 0) {
+                return 0;
+        }
+
+        # If there's already an entity with any of the IPIs, not an autoedit
+        my $reused_ipis = $self->c->model('Artist')->find_reused_ipis(
+            $self->data->{new}{ipi_codes}
+        );
```

Comment:
this runs after `initialize` so these should already be set, same below and in Label::Edit
```suggestion
        my $reused_ipis = $self->reused_ipis;
```